dockerfile/anaconda-release/Dockerfile

```diff
@@ -3,7 +3,8 @@
 # This will be based on our existing anaconda-rpm container because it has all the dependencies
 # required to build Anaconda.
 
-FROM quay.io/rhinstaller/anaconda-rpm:rhel-8
+ARG image
```

Comment:
Why not add a default? `ARG image=quay.io/rhinstaller/anaconda-rpm:f36`

Reply:
I would like to get rid of the default values because it gives more work during branching.